### PR TITLE
Dev watch

### PR DESCRIPTION
**Pull request by Hacksore**
This is the first pass of the local development implementation. This is issue #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,10 @@
 	"scripts": {
 		"build": "rollup -c tools/rollup.module.js && rollup -c tools/rollup.browser.js && rollup -c tools/rollup.browser.min.js",
 		"prepare": "npm test && rm -rf build && npm run build",
-		"test": "eslint src/** tools/** && tslint -c tslint.json types/**.ts"
+		"test": "eslint src/** tools/** && tslint -c tslint.json types/**.ts",
+		"dev": "npm-run-all --parallel watch serve",
+		"watch": "rollup -w -c tools/rollup.browser.min.js",
```
Comment (g-plane) [inline package.json]: No need to use minified version because this is just for development. Uglifying JS code will cost time.

Comment (Hacksore) [reply package.json]: @g-plane True, I dd not think about that.

File: .eslintrc.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 module.exports = {
 	"root": true,
 	"env": {
-		"es6": true
+		"es6": true,
+		"browser": true
```
Comment (yushijinhun) [inline .eslintrc.js]: I don't think it's a good idea to add `browser` environment to `/.eslintrc.js`, in which case eslint won't be able to detect errors if we have wrongly used browser APIs in `tools/`.

The solution is to use multi `.eslintrc.js` files (see https://github.com/to2mbn/skinview3d/commit/79167ab0f855beb6942d576735a19d65f0f503e2).

Comment (printempw) [reply .eslintrc.js]: Agree.

**Pull request merged**